validator/glb_external_ip.rego

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package templates.gcp.GLBExternalIpAccessConstraintV1
```

Comment:
Can we package this under gcp?

Maybe GCPGLBExternalIpAccessConstraint, and `gcp_glb_external_ip.rego`? We also need to have a template added in the [templates directory](https://github.com/forseti-security/policy-library/tree/master/policies/templates).